vision.md

```diff
@@ -0,0 +1,115 @@
+	- device logic including firmware and software
+	- primary only tasked to connect one or many devices/types to overall IoT solution
+- Challenges and needs reagarding device enablement:
+	- enable new services and connectivity on the device while keeping stability and robustness (while having limited computing resources)
+	- dealing with certificates, queuing and persisting messages to handle instable connections, 
+	- allowing the device to be managed centrally, to keep it secure and up-to-date (while important to him not always #1 prio to overall initiative/project)
+
+- Special case : embedded dev with OT focus
+	- familiar with PLCs, SCADA systems,
+	- dealing with emerging need for connectivtiy and IIoT 
+	- key concern is security, robusteness and resource efficiency which is usually overruling any “typical” IT solution on the device and implies some kind of custom logic. (e.g. rather closed OS, no dependencies can be installed, very strict certification and QA process , no CI/CD possible, long prodcut lifecycle 10-20 years)
+
+**Summary:**
+
+The persona types adressed by thin-edge.io often have conflicting requirements and views, this itself is addressed by the project technology vision and design principles, allowing thin-edge.io acting as a brindge between the OT and IT world. 
+
+## What persona needs and problems do we address?
+
+**Key problems for users:**
+
+Until now, when developing device software, IoT proejct teams and device builders had to spend a lot of time solving generic challenges such as connecting to a cloud or IoT, dealing with certificates, queuing and persisting messages to handle instable connections or allowing the device to be managed centrally, to keep it secure and up-to-date.
+
+Today, to overcome those challenges customers can either implement all those components themselves, which typical results in complex embedded code, which usually has less Individual, device-specific, custom code which needs to be maintained over the complete device lifecycle
+
+Implementing all those functionalities on has very high complexity.  Furthermore, more and more companies require devices to be more flexible and host additional software which might be needed to offer new digital services to customers., while taking into account non-functional requirements such as low resource footprint, security and robustness.
+
+The combination of those challenges often leads a series of custom developed embedded software that is expensive to maintain and extend. Also, most of the development used to be specific to one cloud or IoT platform. At the same time for more and more use-cases moving logic and analytics to the embedded edge device becomes a must for reasons such latency, security or cost. However, moving, and orchestrating workloads on the Edge used to be very challenging as it also requires a lot of custom logic to be developed on the device, to be able to integrate and support various device management platforms.
```

Comment:
These sentences are somewhat hard to read. I think it would be better to split them up and rephrase them so as not to repeat the previous sentiment. (Or remove them).